radio/src/translations/en.h.txt

```diff
@@ -71,9 +71,9 @@
 #if defined(PCBI6X)
 #define LEN_AUX_SERIAL_MODES                 "\014"
 #if defined(CLI) || defined(DEBUG)
-#define TR_AUX_SERIAL_MODES                  "Debug\0      ""Telem Mirror""SBUS Trainer"
+#define TR_AUX_SERIAL_MODES                  "Debug\0      ""Telem Mirror""FS Hall Gimbal""SBUS Trainer"
```

Comment:
Try to keep it up to 12 characters, LEN_AUX_SERIAL_MODES = \014 is hex value meaning 12 in decimal notation.